Week1/homework/js-exercises/Exercise 7/lengthString.js

```diff
@@ -0,0 +1,4 @@
+'use strict'
+const mySentence = 'Programming is so interesting!';
+const str = mySentence.length;
```

Comment:
I don't think `str` is a good name for this variable. Try to give it a name that more accurately describes what it is.